mendix/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4119b134-c828-4e14-95b5-585bb13d314a",
+  "app_id": "mendix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Market leading low-code IDE for application development",
+    "title": "Mendix",
+    "media": [
+    {
+      "media_type": "image",
+      "caption": "Mendix Studio Pro 9",
+      "image_url": "images/mx9_splash_screen.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Page Editor",
+      "image_url": "images/page-editor.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Domain model with Data Hub",
+      "image_url": "images/domain-model-with-data-hub.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Cloud developer portal view of available plans",
+      "image_url": "images/cp_environments_popup.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Cloud developer portal view of running apps",
+      "image_url": "images/cp_deployment_app_running.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Cloud developer portal view of resource packs",
+      "image_url": "images/cloud_resource_pack_overview.png"
+    }   
+    ],
+    "classifier_tags": [
+      "Supported OS::Windows",
+      "Category::Cloud"
+    ]
+  },
+  "assets": {},
```

Comment:
We require integrations to include an out-of-the-box dashboard. Please create one in your sandbox account and export it as a JSON file. The file can be attached to this PR. This ensures customers don't need to set up their own dashboards and receive immediate value after installing the integration. Refer [here](https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices) for more info. 

Note: Include a screenshot of your dashboard as an image

Reply:
@eho1307 Thanks for the pointer. 
But currently, we don't have a starter template the customers could use. We would need to give it some thought before we can make it available.

So for the first pass could we publish the integration without any out-of-the-box dashboard ?

Reply:
I've added a dashboard and a sample image.